src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -919,4 +933,59 @@ private Predicate<String> getModulePredicate() {
   public RandomNumberGenerator getRandomizer() {
     return this.populationRandom;
   }
+
+  public void reduceNumberOfHospitals() {
+    // +1 Since this includes the header
+    int hospitalNumber = Integer.parseInt(Config.get("verily.limit_hospital_number")) + 1;
+
+    CSVFormat csvFileFormat = CSVFormat.EXCEL;
+
+    // Verily CSV
+    String verilyFile = "src/main/resources/providers/hospitals_verily.csv";
+    File csvFile = new File(verilyFile);
+    if (!(csvFile.exists() && !csvFile.isDirectory())) {
```

Comment:
should this just return? Why not throw an exception?

Reply:
Fixed it.
``` try {
      if (!(csvFile.exists() && !csvFile.isDirectory())) {
        System.out.println("ERROR: File " + verilyFile + "Does not exists.\n");
      }        
    } catch (Exception e) {}
```